src/Authentication/Authentication/Helpers/StringUtil.cs

```diff
@@ -0,0 +1,104 @@
+// ------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------
+
+
+using System;
+using System.Collections;
+using System.Globalization;
+using System.Net;
+using System.Text;
+using Microsoft.Graph.PowerShell.Authentication.Properties;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Graph.PowerShell.Authentication.Helpers
+{
+    internal static class StringUtil
+    {
+        /// <summary>
+        ///     Formats the specified string, using the current culture.
+        /// </summary>
+        /// <param name="formatSpec"></param>
+        /// <param name="args"></param>
+        /// <returns></returns>
+        internal static string FormatCurrentCulture(this string formatSpec, params object[] args)
+        {
+            return string.Format(CultureInfo.CurrentCulture, formatSpec, args);
+        }
+
+        /// <summary>
+        ///     Formats a Dictionary into a UrlEncoded string
+        /// </summary>
+        /// <param name="content"></param>
+        /// <returns></returns>
+        internal static string FormatDictionary(this IDictionary content)
```

Comment:
NIT: If it makes sense to take a dependency on `Microsoft.AspNetCore.WebUtilities`, you can then use `QueryHelpers.AddQueryString`.

https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.webutilities.queryhelpers.addquerystring?view=aspnetcore-3.1

Reply:
@peombwa What are your thoughts on introducing that dependency?

Reply:
@finsharp the library is not shipped with PowerShell, and we may have to ship the dll with our SDK. That means adding an extra 55KB to the SDK. See https://github.com/Azure/azure-powershell/issues/7921. 

We can omit it for now since we have this method as an alternative. Do we see us needing to use other methods in [microsoft.aspnetcore.webutilities](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.webutilities?view=aspnetcore-2.0)?

Reply:
For now I don't think we will need the extra functionality. Will omit it for now till the need arises

Reply:
Thanks for the context @peombwa. I agree on not introducing a dependency in this case.